buddyboss/buddyboss-platform
# Unable to Translate Strings within JavaScript Files

### All Submissions:


### Changes proposed in this Pull Request:
Closes #362.

### How to test the changes in this Pull Request:
1. Translate ever word through Loco Translate
2. Go to send message to another user
3. Click on recipient
4. See text translated

### Proof Screenshots or Video
English:
http://prntscr.com/qi4yiq
http://prntscr.com/qi4zjc
http://prntscr.com/qi4zn0
http://prntscr.com/qi4zow

Deutsch:
http://prntscr.com/qi4z2a
http://prntscr.com/qi4zxo
http://prntscr.com/qi4zzt
http://prntscr.com/qi501q


### Other information:
